src/Microsoft.Data.SqlClient/ref/Microsoft.Data.SqlClient.Batch.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.Data.SqlClient
+{
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
```

Comment:
Documentation is required here. I don't agree with leaving documentation effort for team, it's part of development and basic documentation must be included in the same PR.

Reply:
I write in idiomatic en-gb. I get tired of being nitpicked by people. It's much better that someone with the target langauge write the documentation do it from the start.

Reply:
Even idiomatic en-gb is fine to start with. At a minimum, the docs should be stubbed out in the XML with TODO or "To be added." content. That way reviewers can make suggestions that are easy to merge into the PR.

Reply:
All supressions removed. Doc comments tags added and a few of the most straightforward ones filled in per the patterns from existing docs. The DBBatch base class doc files were lifted from the current runtime docs for those classes and the namespaces changed to be appropriate.